core/src/test/scala/cats/sequence/TraverseHListSuite.scala

```diff
@@ -11,7 +11,7 @@ import cats.derived._
 import org.scalacheck.Prop.forAll
 
 
-class TraverseSuite extends KittensSuite {
+class TraverseHListSuite extends KittensSuite {
```

Comment:
Is the name fine? I needed to disambiguate the test suites and IMO this one was ill named.